docs/templating/templates/view_matcher_reference.md

```diff
@@ -204,3 +205,12 @@ Matches when the object is a product.
 match:
     '@Ibexa\Contracts\ProductCatalog\ViewMatcher\ProductBased\IsProduct': ~
 ```
+
+## LocationBased\RootLocation
```

Comment:
```suggestion
## Product Catalog Root
```

Reply:
This is not consistent with how we describe all the other matchers in the page.

Reply:
I'm aware but I have mixed feeling about `LocationBased\RootLocation` header. 

This format makes sense for view matchers from ibexa/code because e.g. `Identifier\Section` is exactly how you use it, but view matchers from `ibexa/product-catalog` could be only referred via service id.

Reply:
Yes, this apply to all view matcher defined outside of the `ibexa/core`. Separate PR is OK :wink: